### PR TITLE
Fixes 4 mr

### PR DESCRIPTION
**Pull request by ckolbPTB**
some final fixes

### CODE REVIEW

File: notebooks/MR/c_coil_combination.ipynb [hunk 1]
```diff
@@ -1,44 +1,53 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Demonstration of MR reconstruction with CCP PET-MR Software\n",
```
Comment (KrisThielemans) [inline notebooks/MR/c_coil_combination.ipynb:7]:
```suggestion
    "# Demonstration of MR reconstruction with SIRF\n",
```

File: notebooks/MR/c_coil_combination.ipynb [hunk 2]
```diff
@@ -1,44 +1,53 @@
...
+    "Python notebook such as Jupyter. It is organised in 'cells'. Jupyter displays these\n",
+    "cells nicely and allows you to run each cell on its own.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First version: 27th of May 2017  \n",
+    "Updated 10th of April 2019  \n",
```
Comment (KrisThielemans) [inline notebooks/MR/c_coil_combination.ipynb:21]: might want to update this

File: notebooks/MR/c_coil_combination.ipynb [hunk 3]
```diff
@@ -1,44 +1,53 @@
...
+    "Author: Christoph Kolbitsch, Johannes Mayer  \n",
+    "\n",
+    "\n",
+    "CCP PETMR Synergistic Image Reconstruction Framework (SIRF).  \n",
```
Comment (KrisThielemans) [inline notebooks/MR/c_coil_combination.ipynb:25]:
```suggestion
    "CCP SyneRBI Synergistic Image Reconstruction Framework (SIRF).  \n",
```

File: notebooks/MR/c_coil_combination.ipynb [hunk 4]
```diff
@@ -1,44 +1,53 @@
...
+    "Copyright 2015 - 2017 Rutherford Appleton Laboratory STFC.  \n",
+    "Copyright 2015 - 2017 University College London.  \n",
+    "Copyright 2015 - 2017 Physikalisch-Technische Bundesanstalt.  \n",
```
Comment (KrisThielemans) [inline notebooks/MR/c_coil_combination.ipynb:28]:
```suggestion
    "Copyright 2015 - 2017, 2021 Physikalisch-Technische Bundesanstalt.  \n",
```

**Review by KrisThielemans — CHANGES_REQUESTED** [review]

**Comment by ckolbPTB** [comment]
I hope I did not miss any...

**Pull request merged**